server/isomorphic-routing/index.js

```diff
@@ -6,22 +6,32 @@ import { setSection as setSectionMiddlewareFactory } from '../../client/controll
 
 export function serverRouter( expressApp, setUpRoute, section ) {
 	return function( route, ...middlewares ) {
-		expressApp.get(
-			route,
-			setUpRoute,
-			combineMiddlewares(
-				setSectionMiddlewareFactory( section ),
-				...middlewares
-			),
-			serverRender
-		);
+		if ( middlewares.length === 0 && typeof route === 'function' && route.length === 3 ) {
```

Comment:
> Perhaps we could limit this check to `typeof route == 'function'`? For example, do we ever want to end up in the `else` branch if `route` is a func.

`app.get( someMw )` is a shorthand for `app.get( '*', someMw )`, so that should be handled by the `else` branch; unless `someMw` takes 3 args `( err, context, next )` instead of the usual 2 `( context, next )`.